portal-frontend/src/app/features/public/notice-of-intent/public-notice-of-intent.component.ts

```diff
@@ -39,6 +40,9 @@ export class PublicNoticeOfIntentComponent implements OnInit, OnDestroy {
       const { submission, documents, parcels, decisions } = res;
 
       this.submission = submission;
+      if (submission.status.code === SUBMISSION_STATUS.ALC_DECISION) {
+        this.selectedIndex = 2;
```

Comment:
```suggestion
        this.selectedIndex = 1;
```
NOI only has 2 tabs, previous non-public version was set to 1. Also you may want to change from `SUBMISSION_STATUS` to ` NOI_SUBMISSION_STATUS`

Reply:
Fixed! Good call the NOI Submission Status, led to fixing a few other issues